README.md

```diff
@@ -31,25 +31,40 @@ The Javadocs are in the `./javadoc` directory.
 
 ## Relationship to the Iceberg Repository
 
-The `documentation` folder in the [Iceberg repository](https://github.com/apache/iceberg) contains all the markdown docs used by the website.
-The `common` subfolder contains contents used by the landing page.
-The `versioned` subfolder contains the contents different for each version.
+All markdown pages that are part of a versioned released are maintained in the iceberg repository. All pages common across all version
+releases are kept here in the iceberg-docs repo. A few exceptions are the markdown files that can be found in the `format` folder in
+the iceberg repository and contains markdown files that are copied to `./landing-page/content/common/format`.
 
-During each new release, the release manager copies contents from the Iceberg repository to this docs repository and cuts a new version branch.
-Contents under `common` are copied to `./landing-page/contents/common` here,
-and contents under `versioned` are copied to `./docs/contents/docs` here.
-Javadocs generated from the release are copied to the `javadoc` directory.
+`apache/iceberg`
+- The `docs` folder in the [Iceberg repository](https://github.com/apache/iceberg) contains all the markdown docs used by the **versioned** docs site.
+- The `format` folder contains some items that are common across all versions, such as the Iceberg format specification.
+
+`apache/iceberg-docs`
+- The `docs/content/docs` folder is the target folder when copying the docs over during a version release
+- The `landing-page/content/common` folder is where you can find the common markdown files shared across all versions
+
+During each new release, the release manager will:
+1. Create a branch in this repo from main named for the release version
+2. Copy the contents under `docs` in the iceberg repo to `./docs/content/docs` in the release branch
+3. Copy the contents under `format` in the iceberg repo to `./landing-page/content/common/format` in the release branch
```

Comment:
I thought format changes go into main?

Reply:
Absolutely right, updated this to make this the first step and specifying the main branch instead.